openedx/adg/lms/applications/models.py

```diff
@@ -36,6 +45,12 @@ class ApplicationHub(TimeStampedModel):
         default=False, verbose_name=_('Written Application Submitted'),
     )
     submission_date = models.DateField(null=True, blank=True, verbose_name=_('Submission Date'), )
+    course_status = models.CharField(
```

Comment:
Let's make this a little more descriptive by either changing it to `program_courses_status` or to `omni_courses_status` or any other suitable name.